runtime/oti/cfr.h

```diff
@@ -827,6 +827,9 @@ typedef struct J9CfrClassFile {
 #define CFR_ACC_BRIDGE  					0x00000040
 #define CFR_ACC_VOLATILE  					0x00000040
 #define CFR_ACC_TRANSIENT  					0x00000080
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+#define CFR_ACC_VALUE_TYPE					0x00000100
```

Comment:
Is it a problem this value conflicts with CFR_ACC_NATIVE?

Reply:
`CFR_ACC_NATIVE` is for method modifiers whereas `CFR_ACC_VALUE_TYPE` is for class modifiers